### PR TITLE
Enhance JacksonObjectT to return its wrapped value

### PR DESCRIPTION
**Pull request by rayokota**
This is a small enhancement to JacksonObjectT to allow it to return its wrapped value (or itself).  This is useful for having a CEL expr return the same object passed in to the JacksonRegistry, such as when using a CEL expr which may be an identity function under some conditions.

### CODE REVIEW

**Review by dimas-b — COMMENTED** [review]
Thanks for the contribution, @rayokota !

Please see my comment about exceptions below.

File: jackson/src/main/java/org/projectnessie/cel/types/jackson/JacksonObjectT.java [hunk 1]
```diff
@@ -76,6 +76,12 @@ public Val get(Val index) {
 
   @Override
   public <T> T convertToNative(Class<T> typeDesc) {
+    if (typeDesc.isAssignableFrom(value.getClass())) {
+      return (T) value;
+    }
+    if (typeDesc.isAssignableFrom(getClass())) {
+      return (T) this;
+    }
     throw new UnsupportedOperationException();
```
[the remark this answers is not in the document]
Comment (rayokota) [reply jackson/src/main/java/org/projectnessie/cel/types/jackson/JacksonObjectT.java]: Thanks @dimas-b , I've incorporated your feedback.

**Review by dimas-b — APPROVED** [review]

**Comment by dimas-b** [comment]
@snazy : Are you ok with this change?

**Comment by XN137** [comment]
wondering: could we add testcases for those 3 scenarios?

**Comment by rayokota** [comment]
@XN137 , thanks for looking, I added a unit test

**Review by snazy — APPROVED** [review]
LGTM

**Pull request merged**